doc/templates/index.html

```diff
@@ -8,7 +8,7 @@
         <h1 class="sk-landing-header text-white text-monospace">scikit-learn</h1>
         <h4 class="sk-landing-subheader text-white font-italic mb-3">Machine Learning in Python</h4>
         <a class="btn sk-landing-btn mb-1" href="{{ pathto('getting_started') }}" role="button">Getting Started</a>
-        <a class="btn sk-landing-btn mb-1" href="whats_new/v{{ version }}.html" role="button">What's New in {{ release }}</a>
+        <a class="btn sk-landing-btn mb-1" href="{{ pathto(release_highlights) }}" role="button">Release Highlights for {{ release_highlights_version }}</a>
```

Comment:
Maybe we can directly parse the `{{version}}` in jinja and hardcode 

``https://scikit-learn.org/stable/auto_examples/release_highlights/plot_release_highlightsTHE_PARSED_VERSION_HERE.html`` ?

That would avoid using the `html_context` and stuff. Just wondering

Reply:
I wanted to do that, but we have to make sure that the highlights exist for that version. Which means we have to create a placeholder when we are working on the next version.